readme.md

```diff
@@ -10,40 +9,43 @@
             - [Running the Container](#running-the-container)
             - [docker-compose](#docker-compose)
             - [Notes for Running Using docker-compose](#notes-for-running-using-docker-compose)
+        - [FreeNAS / FreeBSD build and run](#FreeNAS-FreeBSD)
     - [Usage](#usage)
     - [Configuration](#configuration)
 
 <!-- markdown-toc end -->
 # MovieNight stream server
-
 [![Build status](https://api.travis-ci.org/zorchenhimer/MovieNight.svg?branch=master)](https://travis-ci.org/zorchenhimer/MovieNight)
 
 This is a single-instance streaming server with chat.  Originally written to
 replace Rabbit as the platform for watching movies with a group of people
 online.
 
 ## Build requirements
-
 - Go 1.13 or newer
 - GNU Make
 
 ### Older Go Versions
-
 You can install a newer version of Go alongside your OS's distribution by
-following the guide here: [https://golang.org/doc/manage-install](https://golang.org/doc/manage-install)
+following the guide here: [https://golang.org/doc/install#extra_versions](https://golang.org/doc/install#extra_versions)
```

Comment:
This change should be reverted.  The correct URL is `https://golang.org/doc/manage-install`.

Reply:
Done